dask/dataframe/backends.py

```diff
@@ -359,6 +359,8 @@ def __sizeof__(self) -> int:
 
 @group_split_dispatch.register((pd.DataFrame, pd.Series, pd.Index))
 def group_split_pandas(df, c, k, ignore_index=False):
+    if is_series_like(c):
+        c = c.values
```

Comment:
Won't the need to do this break dask_cudf's `group_split_dispatch` implementation? If so, can we submit/merge a cudf PR before merging this?

Reply:
This shouldn't break `dask_cudf` since cuDF's `scatter_by_map` supports Series. Running the full test suite of `dask_cudf` passes.